### PR TITLE
feat: allow customize divider style via style props

### PR DESCRIPTION
**Pull request by picodoth**
addressing https://github.com/react-component/menu/issues/168

### CODE REVIEW

**Comment by yesmeck** [comment]
`propTypes`

默认值最好改成 defaultProps

File: src/Divider.jsx [hunk 1]
```diff
@@ -13,7 +13,17 @@ export default class Divider extends React.Component {
...
+
+Divider.propTypes = {
+  className: PropTypes.string,
+  style: PropTypes.object,
+};
+
+Divider.defaultProps = {
+  className: '',
+  style: {},
+};
```
Comment (yesmeck) [inline src/Divider.jsx]: https://github.com/react-component/menu/blob/87a72cd080fad2c2317f937e5e088bd6ba42ff9a/src/Divider.jsx#L5-L13

Comment (picodoth) [reply src/Divider.jsx]: @yesmeck disabled 还需要么？

Comment (yesmeck) [reply src/Divider.jsx]: 有用的 https://github.com/react-component/menu/commit/7dfd30360a5ceae728d0f442e6eff86f792e04eb

Comment (picodoth) [reply src/Divider.jsx]: 我也觉得有用，可是奇怪的是 divider 里都没引用这个 props

**Pull request merged**